pybadges/version.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 The pybadge Authors
+#
```

Comment:
Could we just put __version__ in __init__.py and load it using distutils as per https://www.python.org/dev/peps/pep-0396/#classic-distutils ?

Reply:
I would suggest not to put version in `init.py` as maintainer could miss and it could be published with the wrong version number. More over having a version.py helps users to tack the newer versions easily. So what are your thoughts should we put it in init.py or keep it in version.py ?
The distutils part will have to be changed.

Reply:
I'm fine with version.py so long as __version__ appears in __init__.py (per PEP-396).

Reply:
Done

Reply:
I see that you import it into `__main__.py` but not `__init__.py`

Reply:
Done